packages/app/src/types/history/index.ts

```diff
@@ -7,12 +7,12 @@ export enum OperationType {
   DOWNLOAD_BACKUP = "DOWNLOAD_BACKUP",
   UPLOAD_BACKUP = "UPLOAD_BACKUP",
   RESET_PASSWORD = "RESET_PASSWORD",
-  ADD_VERIFIABLE_CREDENTIAL = "ADD_VERIFIABLE_CREDENTIAL",
+  NEW_VERIFIABLE_CREDENTIAL = "NEW_VERIFIABLE_CREDENTIAL",
   RENAME_VERIFIABLE_CREDENTIAL = "RENAME_VERIFIABLE_CREDENTIAL",
   DELETE_VERIFIABLE_CREDENTIAL = "DELETE_VERIFIABLE_CREDENTIAL",
   DELETE_ALL_VERIFIABLE_CREDENTIALS = "DELETE_ALL_VERIFIABLE_CREDENTIALS",
   REJECT_VERIFIABLE_CREDENTIAL_REQUEST = "REJECT_VERIFIABLE_CREDENTIAL_REQUEST",
-  GENERATE_VERIFIABLE_PRESENTATION = "GENERATE_VERIFIABLE_PRESENTATION",
+  NEW_VERIFIABLE_PRESENTATION = "NEW_VERIFIABLE_PRESENTATION",
```

Comment:
```suggestion
  NEW_VP = "NEW_VP",
```

Reply:
Same as above, I think this should be the full name.